### PR TITLE
check that src is valid before attempting to apply it

### PR DESCRIPTION
**Pull request by moloko**
fixes https://github.com/adaptlearning/adapt_framework/issues/1768

### CODE REVIEW

File: js/adapt-contrib-gmcq.js [hunk 1]
```diff
@@ -48,12 +48,14 @@ define([
...
         resizeImage: function(width) {
-
             var imageWidth = width === 'medium' ? 'small' : width;
 
             this.$('label').each(function(index) {
-                var src = $(this).find('img').attr('data-' + imageWidth);
-                $(this).find('img').attr('src', src);
+                var $img = $(this).find('img');
+                var newSrc = $img.attr('data-' + imageWidth);
+                if(newSrc) {
```
Comment (oliverfoster) [inline js/adapt-contrib-gmcq.js]: space man

Comment (moloko) [reply js/adapt-contrib-gmcq.js]: park in it man?

Comment (oliverfoster) [reply js/adapt-contrib-gmcq.js]: between ``if`` and ``(`` if you could

Comment (moloko) [reply js/adapt-contrib-gmcq.js]: done man

**Pull request merged**